<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>761</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入结束进程ID：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stopPID">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCpu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStopPro">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>结束进程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>内存占用比：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前调度算法：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProSchedule">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FCFS</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPageSchedule">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LRU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前换页算法：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
